src/assets/languages/english.json

```diff
@@ -238,8 +240,32 @@
         "heading": "Prevention",
         "stopTheBreed": {
           "heading": "Eliminate all the mosquito breeding places by clicking over them ",
-          "alerts" : {
-            "title" : "Congratulations!! <br> You eliminated a "
+          "alerts": {
+            "title": "Congratulations!! <br> You eliminated a "
+          }
+        },
+        "stridesAgainst": {
+          "heading": "Click over the man to help moving it. You should reach the final point.",
```

Comment:
Can we have something like 
```
        "heading": "Locked Stage",
        "howtoplay": {
          "heading": "Prevent the mosquito from transmitting the virus to the human",
          "instructions": {
            "title": "How to play?",
            "ins1": "Select the inventory item",
            "ins2": "Use the item to stop the mosquito"
          },
```
See Locked stage heading and how to play instructions,

Reply:
@Raxerz Done the changes.

Reply:
I am not seeing the changes @lunayach . The heading still seems to be the same.

Reply:
replace heading with how to play instructions and it can be something like:
```
        "howtoplay": {
          "heading": "Help this person reach his destination by clearing all the obstacles",
          "instructions": {
            "title": "How to play?",
            "ins1": "Click on the person to make him move",
            "ins2": "Answer the question correctly to clear the obstacle"
          },
```

Reply:
Okay! I misunderstood the changes... Doing them.